index.js

```diff
@@ -1,39 +1,137 @@
+      humanResponse = await ask('You have already picked up all items in this room\n');
+    }  
+    else if (humanResponse.toLowerCase() === "take paper" ||
+    humanResponse.toLowerCase() === "take seven days" ||
+    humanResponse.toLowerCase() === "pick up paper" ||
+    humanResponse.toLowerCase() === "pick up seven days") {
+      pickUpItem(humanInventory, roomInventory);
+      humanResponse = await ask(`You pick up the paper and leaf through it looking for comics 
+      and ignoring the articles, just like everybody else does.\n`);
+    } 
+    else if(humanResponse.toLowerCase() === "i" && !humanInventory.includes('Seven Days')) {
+      humanResponse = await ask('Your Inventory is empty, take something from the room\n');
+    } 
+    else if(humanResponse.toLowerCase() === "inventory" && !humanInventory.includes('Seven Days')) {
+      humanResponse = await ask('Your Inventory is empty, take something from the room\n');
+    }  
+    else if(humanResponse.toLowerCase() === "take inventory" && !humanInventory.includes('Seven Days')) {
+      humanResponse = await ask('Your Inventory is empty, take something from the room\n');
+    } 
+    else if (humanResponse.toLowerCase() === "i" ||
+    humanResponse.toLowerCase() === "inventory" ||
+    humanResponse.toLowerCase() === "take inventory") {
+      humanResponse = await ask(`You are carrying: A copy of ${humanInventory}, Vermont's Alt-Weekly\n`);
+    }
+    else if (humanResponse.toLowerCase() === "drop paper" && humanInventory.includes('Seven Days') ||
+    humanResponse.toLowerCase() === "drop seven days" && humanInventory.includes('Seven Days')) {
+      dropItem(humanInventory, roomInventory);
+      humanResponse = await ask(`you have dropped ${roomInventory} and now you are a litterer\n`);
+    }
+    else if(humanResponse.toLowerCase() === 'drop paper') {
+      humanResponse = await ask('You have nothing to drop.\n');
+    }
+    else if(humanResponse.toLowerCase() === 'drop seven days') {
+      humanResponse = await ask('You have nothing to drop.\n');
+    } 
+    else if(humanResponse.toLowerCase() === 'read paper' && humanInventory.includes('Seven Days')) {   
+      console.log('You have read the Vermonters sacred mystical texts, you have been warped into a dimension of endless maple syrup.\n');
+      enterState('roomThree');
+    }
+    else if(humanResponse.toLowerCase() === 'read seven days' && humanInventory.includes('Seven Days')) {
+      console.log('\nYou have read the Vermonters sacred mystical texts, you have been warped into a dimension of endless maple syrup.\n');
+      enterState('roomThree');
+    } else {
+        humanResponse = await ask("Evaluate your surroundings! Refer to previous directions.\n");
+      }
+    }
+  while(currentState === 'roomThree') {
+    console.log('You are now drowning in delicious maple syrup, and have gone to heaven \u1F64C\n');
+    break;
+    }
   process.exit();
 }
+  
+    
+
```

Comment:
Your game still requires the following:
Going upstairs
A hungry status
A command to talk to josh
A command to interact with the class
Xyzzy command